actors.md

```diff
@@ -688,26 +677,25 @@ func SubmitPost(proofs PoStProof, faults []FaultSet, recovered Bitfield, done Bi
 		TransferFunds(msg.From, msg.Value-feesRequired)
 	}
 
-	if !CheckPostProof(miner.SectorSize, proof, faults) {
+    seed := GetRandFromBlock(self.ProvingPeriodStart + POST_CHALLENGE_TIME)
+    faultSet := self.faultSet
```

Comment:
Nit: `self.` is a Rust idiom, but you're in a `go` code block.

Reply:
@whyrusleeping  and I agreed to use `self` to refer to the current actor in the pseudo code. should document this somewhere

Reply:
`self` is not used consistently below, `miner` appears too.

Reply:
correct, but I am trying to keep changes minimal

Reply:
correct, but I am trying to keep changes minimal

Reply:
cleaned up